packages/eslint-config-godaddy-flow/package.json

```diff
@@ -24,25 +25,17 @@
   "bin": {
     "eslint-godaddy-flow": "bin/eslint-godaddy-flow"
   },
-  "dependencies": {
-    "eslint-config-godaddy": "^4.0.0"
-  },
   "peerDependencies": {
-    "babel-eslint": ">=7.2.1 <11.0.0",
-    "eslint": "^6.1.0",
+    "eslint": ">=6.0.0"
+  },
+  "dependencies": {
+    "babel-eslint": "^10.0.3",
+    "eslint-config-godaddy": "^4.0.0",
     "eslint-plugin-babel": "^5.3.0",
-    "eslint-plugin-flowtype": "^4.2.0",
-    "eslint-plugin-json": "^1.4.0",
-    "eslint-plugin-mocha": "^5.3.0"
```

Comment:
I don't think these can be just removed. I do understand the `peerDependencies` messages might be annoying. This plugin extends from the base: https://github.com/godaddy/javascript/blob/f115b3cb43e40c39c422f8c051dc3d76017d8291/packages/eslint-config-godaddy-flow/index.js#L3, which in turn has https://github.com/godaddy/javascript/blob/f115b3cb43e40c39c422f8c051dc3d76017d8291/packages/eslint-config-godaddy/index.js#L10. Iirc this will just lead to eslint crashing on not finding the modules.

Reply:
These will now installed as regular dependencies, though, so eslint should be able to find all the relevant plugins, no? It doesn't make sense to me that they're _peer_ dependencies; installing our canned config should install all the relevant plugins for you. All other eslint configs I've seen do this.

Reply:
I stand corrected overlooked them here https://github.com/godaddy/javascript/pull/88/files#diff-502dbc4a5275e8ef5e9db3306e47d92cR31